README.md

```diff
@@ -1 +1,43 @@
-# review_my_past_project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+"review_my_past_project"
+> This is my portfolio-page project I completed during the Admission Trial to enter Microverse. I send it on github to request for a review.
+Kindly review the semantic HTML tags, CSS box model.
+
+
+## Built With
+
+- HTML
+- CSS
+- Technologies used
+
+## Getting Started
+
+The link bellow is provided to clone review_my_past_project from GitHub
+git clone https://github.com/OLIPLICHE/review_my_past_project
+
+Clone this in your local repository
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@OLIPLICHE]
+- LinkedIn: [olipliche-paka-mavoungou]
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
```

Comment:
There is too much blank line within sections. do not add blank lines, spaces without a reason. I advise you to clean the blank lines to make your work looks more professional.
Thank you.